Makefile

```diff
@@ -12,35 +12,31 @@ CPU_ARCH ?= amd64
 MANIFEST_NAME := atlassianlabs/$(BINARY_NAME)
 IMAGE_NAME := $(MANIFEST_NAME)-$(CPU_ARCH)
 ARCH ?= $$(uname -s | tr A-Z a-z)
-GOVERSION := 1.17.2  # Keep in sync with .travis.yml and README.md
+GOVERSION := 1.19  # Keep in sync with .travis.yml and README.md
 GP := /gopath
 MAIN_PKG := github.com/atlassian/gostatsd/cmd/gostatsd
 CLUSTER_PKG := github.com/atlassian/gostatsd/cmd/cluster
-PROTOBUF_VERSION ?= 3.6.1
```

Comment:
That is an insane jump...

Reply:
Check the gostatsd.pb.go generated file, it's not what it looks like.